airbyte-connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -213,7 +217,6 @@ components:
                 description: The name of the stream
               url:
                 type: string
-                format: uri
```

Comment:
if we have interpolated urls, depending on the config, it might not be a valid URL, this is our best effort way of not crashing if this happens and just displaying a URL as a string